src/Microsoft.Diagnostics.Monitoring.Options/StorageOptions.cs

```diff
@@ -8,6 +8,11 @@ namespace Microsoft.Diagnostics.Monitoring.WebApi
 {
     internal class StorageOptions
     {
+        [Display(
+            ResourceType = typeof(OptionsDisplayStrings),
+            Description = nameof(OptionsDisplayStrings.DisplayAttributeDescription_StorageOptions_DefaultSharedPath))]
+        public string DefaultSharedPath { get; set; }
```

Comment:
I'll adjust my managed code to use SharedLibraryPath

Reply:
Not `SharedLibraryPath`, you'll use `DefaultSharedPath`

Reply:
At least until yet another property is added for where the profiler should produce files e.g. its communication socket.